aviary/examples/run_aviary_example.py

```diff
@@ -0,0 +1,127 @@
+            "final_altitude": (32000.0, "ft"),
+            "altitude_bounds": ((0.0, 34000.0), "ft"),
+            "throttle_enforcement": "path_constraint",
+            "fix_initial": True,
+            "constrain_final": False,
+            "fix_duration": False,
+            "initial_bounds": ((0.0, 0.0), "min"),
+            "duration_bounds": ((64.0, 192.0), "min"),
+        },
+        "initial_guesses": {"times": ([0, 128], "min")},
+    },
+    "cruise": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "optimize_mach": False,
+            "optimize_altitude": False,
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_range": False,
+            "initial_mach": (0.72, "unitless"),
+            "final_mach": (0.72, "unitless"),
+            "mach_bounds": ((0.7, 0.74), "unitless"),
+            "initial_altitude": (32000.0, "ft"),
+            "final_altitude": (34000.0, "ft"),
+            "altitude_bounds": ((23000.0, 38000.0), "ft"),
+            "throttle_enforcement": "boundary_constraint",
+            "fix_initial": False,
+            "constrain_final": False,
+            "fix_duration": False,
+            "initial_bounds": ((64.0, 192.0), "min"),
+            "duration_bounds": ((56.5, 169.5), "min"),
+        },
+        "initial_guesses": {"times": ([128, 113], "min")},
+    },
+    "descent": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "optimize_mach": False,
+            "optimize_altitude": False,
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_range": False,
+            "initial_mach": (0.72, "unitless"),
+            "final_mach": (0.36, "unitless"),
+            "mach_bounds": ((0.34, 0.74), "unitless"),
+            "initial_altitude": (34000.0, "ft"),
+            "final_altitude": (500.0, "ft"),
+            "altitude_bounds": ((0.0, 38000.0), "ft"),
+            "throttle_enforcement": "path_constraint",
+            "fix_initial": False,
+            "constrain_final": True,
+            "fix_duration": False,
+            "initial_bounds": ((120.5, 361.5), "min"),
+            "duration_bounds": ((29.0, 87.0), "min"),
+        },
+        "initial_guesses": {"times": ([241, 58], "min")},
+    },
+    "post_mission": {
+        "include_landing": False,
+        "constrain_range": True,
+        "target_range": (1906, "nmi"),
+    },
+}
+
+
+prob = av.AviaryProblem(phase_info, "simple", "FLOPS", av.AnalysisScheme.COLLOCATION)
+
+# Load aircraft and options data from user
+# Allow for user overrides here
+prob.load_inputs('models/test_aircraft/aircraft_for_bench_FwFm.csv')
+
+# Have checks for clashing user inputs
+# Raise warnings or errors depending on how clashing the issues are
```

Comment:
Just something I was thinking about, but `check_inputs` does more than what we say in these comments, since it also runs all of the preprocessors.  I don't know what to do with that fact, but there might be some maverick users who decide they don't need checking because the name makes it feel more options.

This is unrelated to this PR, but I thought about it because we have been carrying those 2 comment lines over whenever we copy the interface.

Reply:
Great call Ken. I've changed it to `check_and_preprocess_inputs` and modified the comments surrounding it in all usage.